src/TheThingsNetwork.cpp

```diff
@@ -279,10 +283,6 @@ int TheThingsNetwork::sendBytes(const byte* payload, int length, int port, bool
   }
 
   String response = readLine(10000);
-  if (response == "") {
-    debugPrintLn(F("Time-out"));
-    return -2;
-  }
```

Comment:
If you have `confirm` enabled, then we may have to wait very, very long. 10 seconds is not enough, it may be many minutes. We have to block here before we get a response.